src/Sulu/Bundle/AudienceTargetingBundle/EventListener/AudienceTargetingCacheListener.php

```diff
@@ -42,24 +42,28 @@ class AudienceTargetingCacheListener implements EventSubscriberInterface
 
     public function preHandle(CacheEvent $cacheEvent)
     {
-        if (self::TARGET_GROUP_URL === $cacheEvent->getRequest()->getRequestUri()) {
+        $request = $cacheEvent->getRequest();
+
+        // requests like "/_sulu_target_group" or "/_fos_user_context_hash" should be ignored
```

Comment:
```suggestion
        // the friendsofsymfony/http-cache-bundle package uses the "internalRequest" attribute to mark internal requests
        // return early in this case to prevent loops on requests "/_sulu_target_group" or "/_fos_user_context_hash"
```